test/parse/test_attribute_list_declaration.rb

```diff
@@ -12,7 +12,7 @@ def test_linear_performance_space
       assert_linear_performance(seq, rehearsal: 10) do |n|
         REXML::Document.new("<!DOCTYPE schema SYSTEM \"foo.dtd\" [<!ATTLIST " +
                             " " * n +
-                            " root v CDATA #FIXED \"test\">]>")
+                            " root v CDATA #FIXED \"test\">]><a/>")
```

Comment:
`<!DOCTYPE` uses root element name:

```suggestion
                            " root v CDATA #FIXED \"test\">]><root/>")
```

Reply:
OK, I see.